### ISSUE TITLE
Pet talent system needs a rework to function correct!

### ISSUE THREAD

**Issue by wazerstar** [Lethay/TalentedClassic#99]
Hi so current system in WOTLK it thinks that it has as many talent points as you as character.
https://i.imgur.com/kRtx1XF.png

You can also set your pet to either dps/tank/healer in top right corner
https://i.imgur.com/MJBfGwE.png

You can only have a Max of 20 talents for your hunter pet when you are max level 80 as beast mastery.
https://i.imgur.com/whW3T3M.png

Pets start earning talent points at level 20 and then gain a point every 4 levels. So by the time your hunter reaches level 80, the pet will have 16 spendable points, unless the hunter has the [Beast Mastery](https://wowwiki-archive.fandom.com/wiki/Beast_Mastery) talent, which adds 4 more points.

**Comment by HopelessRomant1c** [comment]
@Lethay I'm a big fan of this addon, but for hunters, having to disable it every time you want to spent your pet's talent points, it's just too big of a drawback.

Is there any hope for this to be fixed?

**Comment by Lethay** [comment]
The strange thing about this is that I fixed this - calculated level, talent positioning and all. I must've introduced another bug that undid this during another update. I wouldn't have noticed as I don't play a hunter.

I will fix this (again), when I play the game. But my /played since reaching L80 is about 5 minutes or so. It might take a wee bit, unless someone else puts in a pull request.

**Comment by dcasez** [comment]
yeah, love the nice large talent trees.  really like the addon.  really wish the pet talent tree thing was fixed as that is a big bummer for hunters.  otherwise, love it.
